File: SDKs/amazonaws.com/transfer/2018-11-05/java/lib/src/main/java/openapisdk/models/shared/WorkflowDetail.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * WorkflowDetail
 * Specifies the workflow ID for the workflow to assign and the execution role used for executing the workflow.
**/
public class WorkflowDetail {
    @JsonProperty("ExecutionRole")
    public String executionRole;
    public WorkflowDetail withExecutionRole(String executionRole) {
        this.executionRole = executionRole;
        return this;
    }
    @JsonProperty("WorkflowId")
    public String workflowId;
    public WorkflowDetail withWorkflowId(String workflowId) {
        this.workflowId = workflowId;
        return this;
    }
}

File: SDKs/amazonaws.com/transfer/2018-11-05/python/src/sdk/models/shared/workflowdetail.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class WorkflowDetail:
    r"""WorkflowDetail
    Specifies the workflow ID for the workflow to assign and the execution role used for executing the workflow.
    """
    
    execution_role: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('ExecutionRole') }})
    workflow_id: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('WorkflowId') }})
    
